api/database/factories/PoolCandidateFactory.php

```diff
@@ -99,4 +100,40 @@ public function suspended()
             ];
         });
     }
+
+    /**
+     * Add screening questions fully from factory, this is split off as there is now a big factory chain
+     * ScreeningQuestionResponse calls ScreeningQuestion which calls Pool which calls Team which calls User
+     * For unit testing purposes where you aren't seeding 100+ pool candidates
+     * @return void
+     */
+    public function addScreeningQuestionResponseFull(int $questionCount = 3)
```

Comment:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/6237/commits/254c7b1481249d8a7166b0f4fceab79f84f7c7cd and https://github.com/GCTC-NTGC/gc-digital-talent/pull/6237/commits/9ee29a6da15c4a6f53a9431b16cbc0a53112d0c2
I did something I don't think we have done much of yet, using cascades. Screening questions only exist within a scope of a pool candidate and so if you want to delete the pool candidate you need to delete the responses too. And deleting responses for a draft application isn't a big deal since it wasn't submitted, so something to handle automatically.

Reply:
Oh, cool, I really like this.  Great solution!